app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -189,11 +195,12 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
                 return
             }
 
-            val currFragment = fragmentManager.findFragmentByTag(SettingsFragment.FRAGMENT_TAG)
```

Comment:
huh, I guess that old code wasn't actually very correct, bc it only checked for the Settings fragment...